Session_1/module_1.py

```diff
@@ -2,35 +2,77 @@
 
 
 def task_1(array: List[int], target: int) -> List[int]:
```

Comment:
In this PR only files for Session 2 should be changed. 
To do it, you should always create new branch from `main` branch (not previous session).